Suko/View Controller/SUKEventDetailsViewController.m

```diff
@@ -9,22 +9,41 @@
 #import <CoreLocation/CoreLocation.h>
 #import <Contacts/CNPostalAddressFormatter.h>
 #import <Parse/PFImageView.h>
+#import "SUKNotCurrentUserProfileViewController.h"
 
 @interface SUKEventDetailsViewController ()
 @property (weak, nonatomic) IBOutlet UILabel *eventNameLabel;
 @property (weak, nonatomic) IBOutlet UILabel *addressLabel;
 @property (weak, nonatomic) IBOutlet UILabel *dateLabel;
 @property (weak, nonatomic) IBOutlet UILabel *usernameLabel;
-@property (weak, nonatomic) IBOutlet PFImageView *userProfileImage;
+@property (weak, nonatomic) IBOutlet PFImageView *profileImageView;
 @property (weak, nonatomic) IBOutlet UILabel *decriptionLabel;
 @property (weak, nonatomic) IBOutlet UIButton *registerButton;
+@property (weak, nonatomic) IBOutlet UIActivityIndicatorView *spinner;
 
 @end
 
 @implementation SUKEventDetailsViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    
+    UITapGestureRecognizer *profileImageTapGestureRecognizer = [[UITapGestureRecognizer alloc]initWithTarget:self action:@selector(didTapUserProfile:)];
+    [self.profileImageView addGestureRecognizer:profileImageTapGestureRecognizer];
+    [self.profileImageView setUserInteractionEnabled:YES];
+    UITapGestureRecognizer *usernameTapGestureRecognizer = [[UITapGestureRecognizer alloc]initWithTarget:self action:@selector(didTapUserProfile:)];
+    [self.usernameLabel setUserInteractionEnabled:YES];
+    [self.usernameLabel addGestureRecognizer:usernameTapGestureRecognizer];
+    
+    self.registerButton.layer.cornerRadius = 4;
+    self.registerButton.layer.masksToBounds = true;
+    
+    self.spinner.hidesWhenStopped = YES;
+    [self.spinner startAnimating];
+}
+
+- (void) didTapUserProfile:(UITapGestureRecognizer *)sender{
+    [self performSegueWithIdentifier:@"EventDetailsToNotCurrentUserProfileSegue" sender:self.event.postedBy];
```

Comment:
Could make @"EventDetailsToNotCurrentUserProfileSegue" a constant at the top of the file.